### PR TITLE
Bugfix for Layer By Time Visibility 

### PR DESCRIPTION
**Pull request by Polaris5514**
### Description of the Change
Visibility of Duplicate nodes was being set to 0, changed to 1

### Alternate Designs
N/A

### Why Should This Be In Core?
Bugfix

### Benefits
All nodes Visible 

### Possible Drawbacks
None realised

### Applicable Issues
#830

### CODE REVIEW

**Review by arcturus2 — CHANGES_REQUESTED** [review]
@Polaris5514 thanks for this and what you have proposed will work. However I was also thinking about just not setting the `vxVisibilityAttr` attribute because it does default to `1`. This would mean that we could delete the definition of `vxVisibilityAttr` and any references to it. If you have tried this already and if that wasn't working then let me know and I'll be happy to approve this.

**Comment by Polaris5514** [comment]
@arcturus2 Apologies, I had attempted this yesterday but Node 0 was still invisible. Have now found that Node 0 is set to invisible on creation of the sphere graph, not within the Layer By Time plugin. 
Requested changes have now been properly implemented

File: CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/time/LayerByTimePlugin.java [hunk 1]
```diff
@@ -441,7 +437,6 @@ public void read(final GraphReadMethods rg, final PluginInteraction interaction,
                 //Tag on the back of this loop and update the visibility of each node to match it's z value
                 if (isTransactionLayers) {
                     float vxZ = wgcopy.getFloatValue(zAttr, vertexId);
-                    wgcopy.setFloatValue(vxVisibilityAttr, vertexId, vxZ / (z + 1));
                 }
```
Comment (arcturus2) [inline CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/time/LayerByTimePlugin.java:440]: @Polaris5514 from my reading of the code I think this entire if block could be removed. What do you think?

**Review by arcturus2 — CHANGES_REQUESTED** [review]
All good @Polaris5514, you have nothing to apologies for. I think I've found a bit more code that could be deleted so let me know when you have a chance to review.

The advantage of removing code is that it potentially can uncover additional redundant code and takes a step towards removing technical debt which is a good thing 😄

**Comment by Polaris5514** [comment]
Thanks @arcturus2, I have removed the suggested if block. Have also looked through and found two more redundant sections of code and removed them as well.

File: CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/time/LayerByTimePlugin.java [hunk 2]
```diff
@@ -356,8 +355,6 @@ public void read(final GraphReadMethods rg, final PluginInteraction interaction,
             final int timeLayerAttr = wgcopy.addAttribute(GraphElementType.TRANSACTION, LayerNameAttributeDescription.ATTRIBUTE_NAME, LAYER_NAME, "time layer", defaultName, null);
             wgcopy.addAttribute(GraphElementType.GRAPH, IntegerAttributeDescription.ATTRIBUTE_NAME, NLAYERS, "The number of layers to layer by time", 1, null);
             final int zAttr = wgcopy.addAttribute(GraphElementType.VERTEX, FloatAttributeDescription.ATTRIBUTE_NAME, "z", "z", 0, null);
```
Comment (antares1470) [inline CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/time/LayerByTimePlugin.java:357]: this variable is apparently no longer being used. If you're able to remove it that would be great.

**Review by antares1470 — CHANGES_REQUESTED** [review]

**Review by antares1470 — APPROVED** [review]

**Pull request merged**